### PR TITLE
Controller spec

### PR DESCRIPTION
**Pull request by AckonSamuel**
### Project requirements

- [x] - Add RSpec as a gem to your project.
- [x] - Create a Request spec file for all your controllers.
- [x] - Make sure that for each action you check:
- [x]     - If response status was correct.
- [x]     - If a correct template was rendered.
- [x]     - If the response body includes correct placeholder text.

### CODE REVIEW

File: spec/request/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    it 'Checks whether it renders the correct template' do
+      expect(respnse).to render_template(:show)
+    end
+
+    it 'Checks presence of expected placeholder text' do
+      expect.call(response.body).to include('<h2>users display</h2>')
+    end
+  end
+end
```
Comment (GzimAsani) [inline spec/request/posts_spec.rb:38]: - [ ] You need to have a folder and tests for `users_spec.rb` as well

Comment (GzimAsani) [inline spec/request/posts_spec.rb:38]: - [ ] Your tests are failing, kindly check into and update them
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello    @AckonSamuel       :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the tests :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @AckonSamuel, 

Well done making the required changes 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**